modules/utility/functions/zsh-help

```diff
@@ -0,0 +1,59 @@
+# Function to search zsh's man pages in a convenient way
+function zsh-help {
```

Comment:
Because this will be an autoloaded function, the file should either call `zsh-help "$@"` at the end or comment out the function declaration (and closing brace).

See `modules/utility/functions/diff` for an example of the former and `modules/utility/functions/prep` for the latter.

I don't think we've decided on a specific format for prezto, so either should be fine for now.